torch/csrc/jit/codegen/cuda/index_compute.cpp

```diff
@@ -260,7 +260,7 @@ void updateHaloInfoForReference(
     auto consumer_root_axis = *consumer_it;
     auto root_axis_info =
         gpu_lower->haloInfo().getRootAxisInfo(consumer_root_axis);
-    if (root_axis_info.width() == 0) {
+    if (root_axis_info.width() == nullptr) {
```

Comment:
Thanks for catching this. The original code is not correct. It should be:

```
if (!root_axis_info.width()->isZeroInt()) {
```

Reply:
Should this be `if (root_axis_info.width()->isZeroInt()) {` instead?
The other one seems to be breaking cpp tests.

Reply:
My bad, you're right.